authentication/templates/authentication/register.html

```diff
@@ -0,0 +1,153 @@
+                    <a class="text-decoration-none" href="{% url 'authentication:register-view' %}"><b>Register</b></a>
+                </div>
+            </div>
+            <form class="" action="javascript:register()" method="post" enctype="multipart/form-data">
+                <div class="input-group my-2">
+                    <span class="input-group-text bg-primary" id="basic-addon1">
+                        <i class="fas fa-user text-white"></i>
+                    </span>
+                    <input type="text" class="form-control" name="username" placeholder="Username" id="username"
+                        required>
+                </div>
+
+                <div class="text-center text-danger my-3" id="msg-username"></div>
+
+                <div class="input-group my-3">
+                    <span class="input-group-text bg-primary" id="basic-addon1">
+                        <i class="fas fa-lock text-white"></i>
+                    </span>
+                    <input class="form-control" type="password" name="password" placeholder="Password" id="password"
+                        required>
+                </div>
+
+                <div class="text-center text-danger my-3" id="msg-password"></div>
+
+                <div class="input-group my-3">
+                    <span class="input-group-text bg-primary" id="basic-addon1">
+                        <i class="fas fa-lock text-white"></i>
+                    </span>
+                    <input class="form-control" type="password" name="confirm-password" placeholder="Confirm Password"
+                        id="confirm-password" required>
+                </div>
+
+                <div class="text-center text-danger my-3" id="msg-confirm-password"></div>
+                <input type="submit" class="form-control fw-bold btn btn-primary" value="Register"
+                    onclick="register()"></input>
+                <div class="text-center text-success my-3" id="msg-success"></div>
+                <div class="text-center text-danger my-3" id="msg-password-match"></div>
+            </form>
+
+        </div>
+    </div>
+</body>
+
+<script>
+
+    async function login(data) {
+        //debugger
+        let response = await fetch("{% url 'authentication:create-token' %}", {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json'
+            },
+            body: JSON.stringify(data)
+        });
+        data = await response.json();
+
+        if (response.ok) {
+            sessionStorage.setItem("access", data.access)
+            sessionStorage.setItem("refresh", data.refresh)
+            sessionStorage.setItem("user_id", data.user_id)
+            window.location.href = "{% url 'core:home-view' %}";
+
+        } else {
+            document.getElementById('msg').innerHTML = data.detail;
+        }
+    }
+
+    function clear_all_msg() {
```

Comment:
Please use camelCase in JS
```suggestion
    function clearAllMsg() {
```
Though this function name is also a bit confusing/not much clear.